README.md

```diff
@@ -28,11 +28,12 @@ Additionally we're offering 2 special tag suffixes:
 We're also offering special tags for specific PHP versions, e.g. `php8.2.5-v2.0`. 
 
 ### Pimcore version compatibility & recommendations
-| Image / Pimcore | v10 | v11 | v2023.0 |
-|-----------------|-----|----|--------|
-| v1              | ✅   | ✅  | ✅      |
-| v2              | ❌   | ✅  | ✅      |
-| v3              | ❌   | ✅* | ✅*     |
+| Image / Pimcore | v10 | v11 | v2023.x |v2024.x |v2025.x
+|-----------------|-----|--|---------|--------|------|
+| v1              | ✅   | ✅ | ✅       |✅*   |✅*   |
```

Comment:
Isn't `v1` only PHP `8.0` and `8.1`? So it's probably not compatible with Pimcore `v2025.x`? Or is `v2025.x` not Pimcore 12?